### PR TITLE
[RayService] Change runtime env for e2e autoscaling test

### PR DESCRIPTION
**Pull request by zcin**
## Why are these changes needed?

Change the repo used for the runtime env for the autoscaling e2e test to [ray-project/test_dag](https://github.com/ray-project/test_dag/tree/master).

## Checks

- [ ] I've made sure the tests are passing. 
- Testing Strategy
   - [ ] Unit tests
   - [ ] Manual tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: ray-operator/config/samples/ray-service.autoscaler.yaml [hunk 1]
```diff
@@ -17,10 +17,10 @@ spec:
   serveConfigV2: |
     applications:
       - name: app1
-        import_path: autoscaling_test.blocked.app
+        import_path: autoscaling.blocked.app
         route_prefix: /
         runtime_env:
-          working_dir: "https://github.com/ray-project/serve_workloads/archive/7a0daafdb86b55118534b04d122b1b79c9ae99f5.zip"
+          working_dir: "https://github.com/ray-project/test_dag/archive/7a0daafdb86b55118534b04d122b1b79c9ae99f5.zip"
```
Comment (kevin85421) [inline ray-operator/config/samples/ray-service.autoscaler.yaml:23]: In my understanding, the format of the URL is `https://github.com/ray-project/${REPOSITORY}/archive/${COMMIT_OR_BRANCH}.zip`. The test_dag repository may not have the commit `7a0daafdb86b55118534b04d122b1b79c9ae99f5`. Perhaps we can consider using a branch instead, such as https://github.com/ray-project/test_dag/archive/master.zip.

File: ray-operator/config/samples/ray-service.autoscaler.yaml [hunk 2]
```diff
@@ -17,10 +17,10 @@ spec:
   serveConfigV2: |
     applications:
       - name: app1
-        import_path: autoscaling_test.blocked.app
+        import_path: autoscaling.blocked.app
```
Comment (kevin85421) [inline ray-operator/config/samples/ray-service.autoscaler.yaml:20]: https://github.com/ray-project/kuberay/pull/1175#discussion_r1236013964

```suggestion
        import_path: autoscaling.blocked:app
```

File: ray-operator/config/samples/ray-service.autoscaler.yaml [hunk 3]
```diff
@@ -35,10 +35,10 @@ spec:
             ray_actor_options:
               num_cpus: 0.5
       - name: signal
-        import_path: autoscaling_test.signaling.app
+        import_path: autoscaling.signaling.app
```
Comment (kevin85421) [inline ray-operator/config/samples/ray-service.autoscaler.yaml:38]:
```suggestion
        import_path: autoscaling.signaling:app
```

File: ray-operator/config/samples/ray-service.autoscaler.yaml [hunk 4]
```diff
@@ -35,10 +35,10 @@ spec:
...
         route_prefix: /signal
         runtime_env:
-          working_dir: "https://github.com/ray-project/serve_workloads/archive/7a0daafdb86b55118534b04d122b1b79c9ae99f5.zip"
+          working_dir: "https://github.com/ray-project/test_dag/archive/7a0daafdb86b55118534b04d122b1b79c9ae99f5.zip"
```
Comment (kevin85421) [inline ray-operator/config/samples/ray-service.autoscaler.yaml:41]: Same

**Review by kevin85421 — APPROVED** [review]
LGTM. Would you mind testing it manually? Thanks!

**Comment by zcin** [comment]
> LGTM. Would you mind testing it manually? Thanks!

Yeah this has been tested manually.

**Review by shrekris-anyscale — APPROVED** [review]
Awesome, thanks for moving the files!

**Comment by kevin85421** [comment]
This YAML file is not included in CI. Merge it.

**Pull request merged**